eventmesh-connector-plugin/eventmesh-connector-pulsar/src/main/java/org/apache/eventmesh/connector/pulsar/config/ClientConfiguration.java

```diff
@@ -36,4 +36,7 @@ public class ClientConfiguration {
 
     @ConfigFiled(field = "authParams")
     private String authParams;
+
+    @ConfigFiled(field = "topicPrefix")
+    private String topicPrefix;
 }
```

Comment:
It may be better to add some descriptive information, such as application scenarios.

Reply:
Because the full format of topic needs a prefix, but the prefix cannot be passed in the url when the topic is carried